src/modules/auth/auth-user.ts

```diff
@@ -1,18 +1,19 @@
 import { Audio, Screen } from '../root/entities';
 import { LightsGroup } from '../lights/entities';
 import SubscribeEntity from '../root/entities/subscribe-entity';
+import { SecurityGroup } from '../../helpers/security';
 
-export interface User {
+export interface AuthUser {
```

Comment:
The default `User` type of express is now explicitly extended with `AuthUser`. To avoid any confusion on which type should be used, I explicitly wanted to use a different name for our self-defined `User`.

Reply:
Makes sense.